frontend/public/components/dashboard/dashboard-card/card-link.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import { Button, OverlayTrigger, Popover } from 'patternfly-react';
+
+const DashboardCardButtonLink: React.FC<DashboardCardButtonLinkProps> = React.memo(({ children, ...rest }) => (
+  <Button bsStyle="link" className="co-dashboard-card__link" {...rest}>{children}</Button>
+));
+
+export const DashboardCardLink: React.FC<DashboardCardLinkProps> = React.memo(({ children, to }) => (
+  <Link to={to} className="co-dashboard-card__link">
```

Comment:
Did you intend to have the same class on the link and the button?

Reply:
Yes, thats on purpose. I need to set `line-height: 1; padding: 0;` on both otherwise it will make the Card header height to big

Reply:
IMO it would be better to make these two separate classes even if they share the same rules. Per BEM, these are different elements.

Reply:
Ok, I wasn't sure which approach is better. Will use different classes then.

Reply:
done